core/src/processing/awt/PSurfaceAWT.java

```diff
@@ -583,6 +583,15 @@ public void setResizable(boolean resizable) {
     }
   }
 
+    /** Set the minimum size of frame */
+  @Override
+  public void setMinimumSize(int x, int y) {
```

Comment:
Use unit instead of int

Reply:
unit? What type is this?

Reply:
Oops, I meant uint.

Reply:
I was looking at other methods and none uses the type that you propose. Why should I use the type `uint` you propose? Other methods use int, like: setLocation, setCursor, setSize. And a make a research and I don't find this type.